pkg/azureclients/vmasclient/interface.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vmasclient
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2020-12-01/compute"
+	"sigs.k8s.io/cloud-provider-azure/pkg/retry"
+)
+
+const (
+	// APIVersion is the API version for VMAS.
+	APIVersion = "2020-12-01"
```

Comment:
We are still using `APIVersion = "2020-06-30"` for other compute APIs, though the data model has been updated to 2020-12-01 (See #541 for details)

Reply:
you mean we keep 2020-06-30 in this pr and let #541 update it to 2020-12-01?

Reply:
yes

Reply:
done